docs/specs/offline_store_format.md

```diff
@@ -0,0 +1,78 @@
+# Feast Offline Store Format
+
+## Overview
+This document describes feature data storage format for offline retrieval in Feast.
+
+One of the design goals of Feast is being able to plug seamlessly into existing infrastructure, and avoiding adding operational overhead to your ML stack. So instead of being yet another database, Feast relies on existing data storage facilities to store offline feature data.
+
+Feast provides first class support for the following data warehouses (DWH) to store feature data offline out of the box:
+* [BigQuery](https://cloud.google.com/bigquery)
+* [Snowflake](https://www.snowflake.com/) (Coming Soon)
+* [Redshift](https://aws.amazon.com/redshift/) (Coming Soon)
+
+The integration between Feast and the DWH is highly configurable, but at the same time there are some non-configurable implications and assumptions that Feast imposes on table schemas and mapping between database-native types and Feast type system. This is what this document is about.
+
+## Terminology
+For brevity, below we'll use just "DWH" for the data warehouse that is used as an offline storage engine for feature data.
+
+For common Feast terms, like "Feature Table", "Entity" please refer to [Feast glossary](https://github.com/feast-dev/feast/blob/master/docs/concepts/glossary.md).
+
+## Table schema
```

Comment:
Should we add information on why these columns are necessary? 
* Feature Tables are by nature time series
* Created timestamps are used to deduplicate feature values with the same event timestamp and entity key